integration/setup/setup.go

```diff
@@ -274,6 +274,18 @@ func installResources(ctx context.Context, config Config) error {
 		}
 	}
 
+	var sshUserList []providerv1alpha1.ClusterKubernetesSSHUser
+	{
+		if env.SSHPublicKey() != "" {
```

Comment:
There was a problem when not passing the SSH public key. Now we only set the SSH public key if something is passed.